osclib/accept_command.py

```diff
@@ -32,6 +32,39 @@ def find_new_requests(self, project):
             rqs.append({'id': int(rq.get('id')), 'packages': pkgs})
         return rqs
 
+    def virtual_accept_request_has_no_binary(self, project, package):
+        pkgs = []
+        filelist = self.api.get_filelist_for_package(pkgname=package, project=self.api.project, expand='1', extension='spec')
+        for spec in filelist:
+            pkgs.append(spec[:-5])
```

Comment:
Since this same sort of logic is done in #1110, perhaps extracting as another method in StagingAPI and using for both would make sense (ie a method to return list of subpackages based on spec files in main package). Might as well merge the other and refactor in this request along with all the additional usages.